src/GraphQL/SortBlockMutationCreator.php

```diff
@@ -1,14 +1,15 @@
 <?php
 namespace DNADesign\Elemental\GraphQL;
 
-use DNADesign\Elemental\Models\BaseElement;
-use GraphQL\Type\Definition\ResolveInfo;
-use GraphQL\Type\Definition\Type;
 use SilverStripe\Core\Convert;
-use SilverStripe\GraphQL\MutationCreator;
-use SilverStripe\GraphQL\OperationResolver;
 use SilverStripe\ORM\DataObject;
+use GraphQL\Type\Definition\Type;
 use SilverStripe\ORM\Queries\SQLUpdate;
+use GraphQL\Type\Definition\ResolveInfo;
+use SilverStripe\GraphQL\MutationCreator;
+use DNADesign\Elemental\Models\BaseElement;
+use SilverStripe\GraphQL\OperationResolver;
+use DNADesign\Elemental\Services\ReorderElements;
```

Comment:
As an aside, it'd be cool if your IDE could sort these alphabetically to reduce noise in future PRs from people whose IDEs do sort them

Reply:
Interesting note: I have sorted them.
But weirdly this is the order. Rather than alphabetically (which I also expected) it seems to go in triangle form, shortest to longest... which is just... weird.

Reply:
OK sweet, there's a setting to make it sort alphabetically. But it does it by FQCN, not just classname... I hope that's OK :)